landing_page_backend/landing_page_backend/settings.py

```diff
@@ -35,8 +35,9 @@
     'django.contrib.sessions',
```

Comment:
Database should be gitignored, as it could contain a sensitive data, like private user data.